src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/page/customfooterlibs.html

```diff
@@ -29,3 +29,11 @@
 ></sly>
 #end
 
+#if ($includeForms == "y" or $includeFormsenrollment == "y" or $includeFormscommunications == "y" or $includeFormsheadless == "y")
+<sly data-sly-use.clientlib="core/wcm/components/commons/v1/templates/clientlib.html"
+     data-sly-use.formstructparser="com.adobe.cq.forms.core.components.models.form.FormStructureParser"
```

Comment:
this will require a new release of core components being embedded in the archetype. lets update the cc version also in this PR else it will fail.

Reply:
@barshat7 Please check @ujjwal5 changes. Your PR is dependent on the same